projects/ng-components/lib/tooltip/tooltip-content/tooltip-content.component.ts

```diff
@@ -23,20 +12,17 @@ export type TooltipTheme = "default" | "light" | "primary" | "warning" | "succes
     styleUrls: ["./tooltip-content.component.scss"],
     animations: [
         trigger("fadeOut", [
-            transition(":enter", [style({ opacity: 0 }), animate(300, style({ opacity: 1 }))]),
-            transition(":leave", [animate(300, style({ opacity: 0 }))]),
+            transition(":enter", [style({ opacity: 0 }), animate(100, style({ opacity: 1 }))]),
+            transition(":leave", [animate(100, style({ opacity: 0 }))]),
```

Comment:
This looks like a very simple animation that can be easily done with CSS. I don't think we need a library for it.

Reply:
i believe this is a very convenient solution, because it offers the possibility to create animation without having to add an extra variable to the component just to append a class ".show", the animation library is smart enough to know that a component is appearing or disappearing .
https://bundlephobia.com/result?p=@angular/animations@10.0.12
![image](https://user-images.githubusercontent.com/14928743/91315028-c531e880-e7e9-11ea-8219-62397e7745b4.png)